include/zephyr/fs/fs_interface.h

```diff
@@ -13,7 +13,7 @@
 extern "C" {
 #endif
 
-#if (CONFIG_FILE_SYSTEM_MAX_FILE_NAME - 0) > 0
+#if defined(CONFIG_FILE_SYSTEM_MAX_FILE_NAME) &&  (CONFIG_FILE_SYSTEM_MAX_FILE_NAME - 0)> 0
```

Comment:
Are you planning to make upstream patch for this?

Reply:
Yes, I'm going to close this one and make upstream pr instead.